backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/FieldData.java

```diff
@@ -13,7 +13,10 @@
 
 /**
  * Utility methods, similar to Lucene's {@link DocValues}.
- */
+ * <p>
+ * Copied and adapted from {@code PUT THE ORIGINAL CLASS NAME HERE}
```

Comment:
I have removed "FieldData.java". I use a different solution. Everything is "DoubleMultiValuesSource" and "LongMultiValuesSource". This can sort and aggregate.